### DISCUSSION TITLE
Could we have a confirmation that the video recording is not "real-time"?

### DISCUSSION THREAD

**Discussion by Songyu-Wang** [cypress-io/cypress#17605] [General]
Hello,

After sharing some Cypress videos with non-tech team members. They are puzzled with the "interrupted" style of video recording. 
Using https://dashboard.cypress.io/projects/7s5okt/runs/4817/test-results/7bed2169-864a-4fab-8be4-ecff46a7a33f/video as an example. If you pay close attention to the timer, the number displayed is "jumpy" instead of matching video frame timestamp 100%. 
I totally understand why this is the case. Reusing the frames reduce the file size. However, I cannot find any Cypress document to discuss this. Therefore it is hard for me to explain to my co-worker this is not an error but by design. 

If it is possible, could anyone from Cypress org to just confirm this is indeed the case as my supportive document. If you think this is something worth calling out, I am happy to create a issue for the document repo

**Comment by Songyu-Wang** [comment]
Seems like lack of resource has something to do with this issue. What I said above may not be true. Since no one cares about this topic, I am gonna consider it "answered"

**Comment by jennifer-shehane** [comment]
Jumpy videos usually indicate a lack of enough resources to run Cypress, any background process, and the video recording. We suggest increasing the resources of the machine.
